.github/workflows/integration.yml

```
@@ -0,0 +1,29 @@
+name: Integration Test
+on: [push, pull_request]
+jobs:
+  test:
+    name: Run integration test
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Should this happen on 3.7, since that's what data-importer is set to?

---

I think the data importer version should get bumped up instead!

---

Would it make any sense to matrix both versions for now, e.g. 
```suggestion
        python-version: [3.8, 3.7]
```

---

(This way we cover ourselves from introducing breakage into importer while preparing the dependencies for the glorious future.)